src/MatrixClientPeg.js

```diff
@@ -245,6 +245,7 @@ class MatrixClientPeg {
 }
 
 if (!global.mxMatrixClientPeg) {
-    global.mxMatrixClientPeg = new MatrixClientPeg();
+    global.mxMatrixClientPeg = new _MatrixClientPeg();
 }
 export default global.mxMatrixClientPeg;
+export const MatrixClientPeg = global.mxMatrixClientPeg;
```

Comment:
Hmm, why are we exporting it as both the default export and a named export?

Reply:
Because exports changed slightly from upgrading babel. The parts of the app that use the require syntax are broken without this, irritatingly.

Reply:
Ah, gotcha, but you should put this in a comment rather than telling me, else be asked the same question by every person who reads this code in the future. ;)

Reply:
can do. I'm about to embark on a journey through every single export we have and will likely come back to clean this one up.